### PR TITLE
Xrddev 389 internal servers

### PR DESCRIPTION
**Pull request by Riippi**
## Description
- Internal servers tab for member and subsystem
- View for certificate info, add, delete and export certificate
- Localisation system

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have followed the agreed [Version Control Practices](https://confluence.niis.org/pages/viewpage.action?spaceKey=XRDDEV&title=Version+Control+Practices)
- [x] My changes generate no new warnings or errors (e.g. Javascript console, Java stdout)
- [ ] I have made corresponding changes to the documentation
- [x] The new code has sufficient test coverage
- [x] The build, unit and integration tests pass
- [x] There is a link to a successful Jenkins build
- [x] No new npm audit issues, or new issues have been added to [Front-end build process](https://confluence.niis.org/display/XRDDEV/Front-end+build+process) tracking table and accepted
- [ ] New backlog items that have been created (such as "not implementing this acceptance criteria now") are mentioned + linked in Jira task comments
- [x] All task outputs (PR, documentation, UI design, etc) is listed in a Jira comment so that it is easy for reviewer to check
- [ ] Deviations from acceptance criteria listed in comments (also if "this criteria was removed")

Jenkins build for the branch https://jenkins.niis.org/view/api-based-ui/job/rest-ui-build-feature-branch/26/console

### CODE REVIEW

File: src/proxy-ui-api/frontend/src/store/modules/client.ts [hunk 1]
```diff
@@ -154,48 +153,64 @@ export const actions: ActionTree<ClientState, RootState> = {
...
+      // Create a link to DOM and click it. This will trigger the browser to start file download.
+      const link = document.createElement('a');
+      link.href = window.URL.createObjectURL(blob);
+      link.setAttribute('download', effectiveFileName);
+      document.body.appendChild(link);
+      link.click();
 
     }).catch((response) => {
-      console.error('Could not Download the Excel report from the backend.', response);
+      console.error('Could not download the certificate', response);
```
Comment (carohauta) [inline src/proxy-ui-api/frontend/src/store/modules/client.ts]: Should this be shown in the snackbar/toast/whatever?

File: src/proxy-ui-api/frontend/src/views/AddSubsystem.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
-  <v-container>
-    <v-layout text-xs-center wrap column class="outer-wrapper">
+  <v-container class="xr-tab-max-width">
+    <v-layout text-xs-center wrap column>
       <v-flex mb-4>
         <h1 class="display-1 font-weight-bold mb-3">Add a Subsystem</h1>
```
Comment (carohauta) [inline src/proxy-ui-api/frontend/src/views/AddSubsystem.vue]: Missing i18n. Or is it still WIP?

File: src/proxy-ui-api/frontend/src/views/Base.vue [hunk 3]
```diff
@@ -2,19 +2,22 @@
...
 
     <v-dialog v-model="logoutDialog" width="500" lazy persistent>
       <v-card class="xroad-card">
         <v-card-title>
           <span class="headline">Session expired</span>
         </v-card-title>
-        <v-card-text class="pt-4">You have been idle for 30 minutes and your session has expired. For security reasons, you will be logged out.</v-card-text>
+        <v-card-text
+          class="pt-4"
+        >You have been idle for 30 minutes and your session has expired. For security reasons, you will be logged out.</v-card-text>
```
Comment (carohauta) [inline src/proxy-ui-api/frontend/src/views/Base.vue]: Also i18n

File: src/proxy-ui-api/frontend/src/components/CertificateLine.vue [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    lineBreaks(value: string) {
+      // Add line break after every 60 characters
+      return value.replace(/(.{60})/g, '$1\n');
+    },
+  },
+  data() {
+    return {};
+  },
+  computed: {},
+  methods: {
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/components/CertificateLine.vue]: Fixed

File: src/proxy-ui-api/frontend/src/components/CertificateLine.vue [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+      if (this.date) {
+        // Return readable presentation of date
+        const event = new Date(this.sourceObject.not_before);
+        return event.toString();
+      }
+
+      return this.sourceObject[this.childKey];
+    },
+
+    formatArray(arr: []) {
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/components/CertificateLine.vue]: Fixed

File: src/proxy-ui-api/frontend/src/components/InternalServers.vue [hunk 6]
```diff
@@ -108,39 +124,85 @@ export default Vue.extend({
...
+    },
   },
   created() {
     this.fetchSSCertificate(this.id);
     this.fetchTlsCertificates(this.id);
   },
   methods: {
-    onFileChange(e: any) {
-      const fileList = e.target.files || e.dataTransfer.files;
+    onFileChange(event: any) {
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/components/InternalServers.vue]: Fixed

File: src/proxy-ui-api/frontend/src/views/AddSubsystem.vue (hunk 2 above)
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/views/AddSubsystem.vue]: WIP. There will be a special PR for all current translations soon.

File: src/proxy-ui-api/frontend/src/views/Base.vue (hunk 3 above)
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/views/Base.vue]: WIP. There will be a special PR for all current translations soon.

**Pull request merged**